papers/optional-ref.md

```diff
@@ -59,14 +61,16 @@ o.transform([&](auto c&){
 :::
 
 # Motivation
-Optionals holding references are common other than in the standard libary's implementation.  The desire for such a feature is well understood, and many optional types in commonly used libraries provide it, with the semanics proposed here.
+Optionals holding references are common other than in the standard libary's implementation (or even in one of the big open sources standard libraries).  The desire for such a feature is well understood, and many optional types in commonly used libraries provide it, with the semanics proposed here.
```

Comment:
I know that libc++ is trying to make sure of ABI stability. But they still `static_assert(!is_reference_v<value_type>` . Is anyone shipping a way for a user to write std::optional<T&> ?

Reply:
not to my knowledge. I was just referring to libc++. We can phrase it in a way to say that libc++ already ships an implementation that is disabled due to the current disallowance of optional<T&> by the standard. Or leave it out.

Reply:
see amended pull request. separate sentence on libc++.